scratch/copy-mk.py

```diff
@@ -185,6 +230,71 @@ def cbf_sensor_value(name: str) -> Any:
         finally:
             client.close()
             await client.wait_closed()
+
+    if args.transfer_delays:
+        stream_names = [f"antenna-channelised-voltage-{input_label}" for input_label in input_labels]
+        # New client because it needs to be initialised with the callback (?)
+        callback_client = KATPortalClient(
+            args.portal,
+            partial(
+                delay_transfer_callback,
+                delays_queue=delays_queue,
+            ),
+        )
+        # Connect before subscribing
+        await callback_client.connect()
+        delay_sensors_regex = (
+            "^(?:"
+            + "|".join(f"{prefix}_wide_{stream_name.replace("-", "_")}_delay" for stream_name in stream_names)
+            + ")$"
+        )
+        # TODO: Check against results returned from the following to
+        # ensure the `callback_client` is ready to proceed.
+        namespace = f"{args.name}_{uuid.uuid4()}"
+        await callback_client.subscribe(namespace)
+        await callback_client.set_sampling_strategies(namespace, delay_sensors_regex, "event")
```

Comment:
I've got something locally to check the result here and print where it failed to subscribe to a sensor.

Should I make the script fail hard here if it can't configure all delay sensors? I genuinely don't know what length/breadth is required for tomorrow's integration test.